### ISSUE TITLE
Sliders won't update when switching from automatic to manual mode

### ISSUE THREAD

**Issue by rverweij** [DynamoDS/Dynamo#5198]
When switching from automatic mode to manual mode, both the integer and number slider don’t seem to update to the value they had when leaving automatic mode, they will update to any other value except that one. 
For instance I have a number slider min=0, max=3, step=1, in automatic mode I set the value to “2” I then switch to manual mode, change the value to something else (no problem) change back to “2” and it won’t update.

![slider issue](https://cloud.githubusercontent.com/assets/14011484/9540746/3e293234-4d62-11e5-9c58-c9c4d4416b76.png)

**Comment by ke-yu** [comment]
hi @rverweij , which version of Dynamo you are using? I'm not able to reproduce it with the latest build.

**Comment by rverweij** [comment]
Just after reporting this here, I switched to Dynamo 0.8.2.2392, I use it in Revit 2015. I think it was the one of the last versions of 0.7 when I noticed this. The problem however still excist in the 0.8.2.2392 version, although now I do get an error message: 

Exception of type 'ProtoCore.Exceptions.CompilerInternalException' was thrown.

   at ProtoCore.Utils.Validity.Assert(Boolean cond)
   at ProtoScript.Runners.ChangeSetApplier.ApplyChangeSetForceExecute(ChangeSetData changeSet)
   at ProtoScript.Runners.LiveRunner.SynchronizeInternal(GraphSyncData syncData)
   at ProtoScript.Runners.LiveRunner.UpdateGraph(GraphSyncData syncData)
   at Dynamo.Core.Threading.AsyncTask.Execute()

Make sure you use the slider in automatic mode first, then switch to manual mode (and remember the slider position upon switching) change the slider position and click run, the slider will update correctly, then change the slider position back to the position which it had upon switching, click run again and you will get an error.
